### DISCUSSION TITLE
Audio boost

### DISCUSSION THREAD

**Discussion by bjohas** [PaulWoitaschek/Voice#1476] [General]
Hi!

I've often used the audio boost feature in previous versions. Is it.planned to being this back for the current series 6?

Previously the setting was limited at "9", but it would be lovely to allow a broader range.

**Comment by techsoulman** [comment]
Hi, I am in complete agreement with bjohas. I frequently run into audiobooks that are too quiet without the boost, especially if I am listening in a loud location.

If you can re-add the feature it would be greatly appreciated!

**Comment by PaulWoitaschek** [comment]
It's already back as of 6.1.1 😊

**Reply by techsoulman** [reply]
Beautiful! Thanks Paul!

This app is too perfect!

**Comment by sthompson-celerity** [comment]
Can we boost above 9? Like to 20? I have some stupid quiet audio books.
